book/website/community-handbook/governance/gov-constitutional.md

```diff
@@ -0,0 +1,131 @@
+
+If the Chair Elect steps down from their role the Steering Committee will elect another member into the role.
+
+If the Chair steps down from their role the Chair Elect will be asked to step into the role of Chair and will serve for the remainder of the Chair's term as well as their full year as Chair.
+
+If the Chair Elect does not want to take on the Chair role earlier than planned then the Steering Committee will elect a Chair from their membership. 
+
+The role of Chair can be shared by up to three Co-Chairs as described in the <!-- MAKE INTO LINK-->role of a chair page in this chapter.
+
+## Ways of working
+
+### Synchronous collaboration
+
+The Steering Council will meet every 3 months online.
+
+They will aim to host their meetings around 1 month before the community forum events to maintain a regular cadance of communication between the Steering Council and the broader community.
+
+It is up to the Steering Council to decide how long each meeting should last.
+
+It is the responsibility of the Chair to set the agenda for these meetings.
+They may choose to - for example - focus on particularly thorny topics for discussion, or to invite presentations on activity across the project to keep all Steering Council members informed.
+
+The Steering Council may also hold ad hoc meetings to discuss specific topics in order to facilitate empowered, transparent and inclusive decision making across the project. 
+
+### Asynchronous collaboration
+
+Steering Council members will be added to a private slack channel in *The Turing Way* slack workspace.
+They can use this space to share knowledge and to ask for input for ongoing discussions.
+
+All requests for decisions by the Steering Council are documented as issues in the governance repository in *The Turing Way* GitHub organisation.
+Steering Council members are expected to monitor these issues and contribute on topics related to the interests and needs of their working group or delivery group members.
+
+### Responding to requests for a decision
+
+Any individual member of The Turing Way community can request a decision from the Steering Council.
+They would make that request in the form of a GitHub issue in the goverance GitHub repository.
+See the page in this chapter on <!-- FORMAT TO LINK --> balancing sensitive information disclosure with transparency for guidance on the appropriate amount of information to include in the public request.
+
+Requests for decisions can also be brought by working groups and delivery groups, and by the Steering Council themselves.
+
+On receiving the request for a decision, the Steering Council may:
+
+1. Approve the recommended action if one is provided.
+2. Develop and endorse a solution to the challenge if no recommended action was proposed.
+3. Develop and endorse an alternative solution to the challenge if they do not agree with the proposed action.
+4. Delegate the decision to a different team.
```

Comment:
Maybe this is missing an option to reject the request?

I like this, feels like RFC.